src/main/java/racingcar/domain/car/Car.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.domain.car;
+
+import java.util.List;
+import racingcar.domain.numberpicker.RandomNumberPicker;
+
+public class Car {
+
+    private static final int MIN_REQUIRED_DIGIT_TO_MOVE_FORWARD = 4;
+
+    private final CarName name;
+    private final ForwardCount forwardCount;
+
+    private Car(CarName name, ForwardCount forwardCount) {
+        this.name = name;
+        this.forwardCount = forwardCount;
+    }
+
+    public static Car nameOf(CarName name) {
+        return new Car(name, ForwardCount.defaultOf());
+    }
+
+    public static Car of(CarName name, ForwardCount forwardCount) {
+        return new Car(name, forwardCount);
+    }
+
+    public static List<Car> namesOf(List<CarName> carNames) {
+        return carNames.stream()
+                .map(Car::nameOf)
+                .toList();
+    }
+
+    public void moveForward(RandomNumberPicker randomNumberPicker) {
+        if (randomNumberPicker.pickOneDigit() >= MIN_REQUIRED_DIGIT_TO_MOVE_FORWARD) {
+            this.forwardCount.increaseByOne();
+        }
+    }
```

Comment:
if 절 안의 가독성을 위해 메소드 분리를 해도 좋을 것 같습니다!

```suggestion
    public void moveForward(RandomNumberPicker randomNumberPicker) {
        if (isOverMoveNumber(randomNumberPicker)) {
            this.forwardCount.increaseByOne();
        }
    }

    private static boolean isOverMoveNumber(RandomNumberPicker randomNumberPicker) {
        return randomNumberPicker.pickOneDigit() >= MIN_REQUIRED_DIGIT_TO_MOVE_FORWARD;
    }
```

Reply:
이렇게 하면 확실히 **이름을 통해 의도를 드러낼** 수 있겠군요! 
저도 이런 리뷰를 많이 남기는 편인데 저 조차도 깜빡할 때가 많네요.. 🤣 꼼꼼한 리뷰 감사합니닷 :D